ReactiveExtensions.playground/Pages/Introduction.xcplaygroundpage/Contents.swift

```diff
@@ -170,21 +170,25 @@ response
  by .01 seconds.
  */
 
-let scheduler = QueueScheduler(queue: dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0))
+let scheduler = QueueScheduler(
+  qos: DispatchQoS.background,
```

Comment:
Shouldn't matter at all if the priority changes here right?

[Seems](https://developer.apple.com/library/content/documentation/Performance/Conceptual/EnergyGuide-iOS/PrioritizeWorkWithQoS.html) `DISPATCH_QUEUE_PRIORITY_HIGH` is now `DispatchQoS.userInitiated`.